### PR TITLE
Association Rules: Output table

### PR DESCRIPTION
**Pull request by ajdapretnar**
##### Issue

Implements #35 

##### Description of changes
- Enable selecting a rule of length one.
- Output a Table of rules.
- Remove some redundant code.

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by kernc — APPROVED** [review]
That's a really good fix for the issue!

**Comment by ajdapretnar** [comment]
Implement is a very much recognized verb in an English language (https://www.merriam-webster.com/dictionary/implement).

File: orangecontrib/associate/widgets/owassociate.py [hunk 1]
```diff
@@ -258,6 +239,35 @@ def filterAcceptsRow(self, row, parent):
...
+            data = []
+            rules = []
+            for row in range(self.rowCount()):
+                data_inst = []
+                rule = []
+                for column in range(self.columnCount()):
+                    index = self.index(row, column)
+                    if column < 6:
+                        data_inst.append(float(self.data(index)))
+                    elif column == 7:
```
[the remark this answers is not in the document]
Comment (ajdapretnar) [reply orangecontrib/associate/widgets/owassociate.py]: Fixed. Also the data creating was streamlined.

**Comment by ajdapretnar** [comment]
@markotoplak Thanks for the help with Travis! ⭐

**Comment by markotoplak** [comment]
@ajdapretnar, thank you for writing the first tests in this add-on. :D

**Pull request merged**